### ISSUE TITLE
Tfgrid: Add example that deploys VMs on the same network

### ISSUE THREAD

**Issue by brandonpille** [threefoldtech/web3_proxy#70]
This will complete the issue https://circles.threefold.me/project/tf_engineering_coordination/task/14: 
_"I can create 2 VM, attach to same network and user webgateway to access webserver as running in VM"_

We should create an example in the examples folder (maybe in a new folder tfgrid) which should do this:

- [ ] Create VMs and attach them all to the same network
- [ ] Allow the user to define how many VMs we want through arguments
- [ ] Allow the user to define the required resources of the VMs through arguments
- [ ] All settings that can and should be chosen by the user should be opened up through arguments

Maybe we should think about passing this configuration through a configuration file (maybe through [markdown docs actions](https://github.com/freeflowuniverse/crystallib/blob/development/markdowndocs/model_action.v)). This example below is the content of a markdown file that contains the configuration of 2 VMs (1 with 2 cpus, the other with 4 and both 4GB ram). This is only a minimal configuration. All required configuration should be able to be defined in this markdown action.
```
!!3bot.tfgrid.vm.deploy
    cpu:2
    memory:4GB
!!3bot.tfgrid.vm.deploy
    cpu:4
    memory:4GB
```
See [markdown docs](https://github.com/freeflowuniverse/crystallib/blob/development/markdowndocs/) on how to use the markdown docs. Come to @brandonpille for any questions.

**Comment by MarioBassem** [comment]
the markdown suggestion is really nice, but i kind of think its redundant here.

when a user writes an action, they specify two things: the action (deploy, update, get, delete), and the needed specs...
if these are the only actions provided, then we could just prepare an example v file that uses the rpc client out of the box for each action and the user would only need to update the specs inside, these specs would not be any different from the ones the user would provide in a markdown file.

if these are not the only actions provided, and we wanted to provide some easy-to-use solutions like, deploying a vm and linking it to some gateawy, then we could also provide an example file to do just that...

so basically, instead of the user having to write an action name in a markdown file, they would just run the right example, the specs are provided in both approaches without any differences. they are basically the same thing, except that in the markdown approach, we would have an extra step of parsing the user's files.

**Comment by MarioBassem** [comment]
as we were adding examples, we encountered this issue with the tfgrid-sdk:
https://github.com/threefoldtech/tfgrid-sdk-go/issues/149

**Comment by DylanVerstraete** [comment]
@MarioBassem what is the status of this one? Are we still adding this in the near future or will this be postponed?

**Comment by MarioBassem** [comment]
this issue consisted of two parts:
- adding the example mentioned
- using a markdown file to provide specs for the deployments

the first part is done, the second part is now 3lang i guess :)

so i believe we can close it as 3lang has its own issue now, no?

**Comment by MarioBassem** [comment]
this is done in #95 
you can now deploy multiple vms on the same network using 3lang or cli.
3lang docs could be found [here](https://github.com/threefoldtech/web3_proxy/blob/development/manual/src/threelang/tfgrid/vm.md)
cli docs could be found [here](https://github.com/threefoldtech/web3_proxy/blob/development/manual/src/examples/tfgrid/vms.md)
